code/src/Components/LoadingSpinner.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import styled from 'styled-components'
+
+const LoadingSpinner = () => {
+    return (
+        <Loader />
+    )
+}
+
+export default LoadingSpinner
+
+const Loader = styled.div`
```

Comment:
Such a long and impressive styling of your LoadingSpinner! Great job with the result!